src/lib/viewers/media/MediaBaseViewer.js

```diff
@@ -60,6 +63,49 @@ class MediaBaseViewer extends BaseViewer {
         this.loadTimeout = 100000;
         this.oldVolume = DEFAULT_VOLUME;
         this.pauseListener = null;
+
+        this.startTimeInSeconds = this.getStartTimeInSeconds();
+    }
+
+    /**
+     * Converts a value and unit to seconds
+     *
+     * @param {string|number} value - the value e.g. 1
+     * @param {string} unit - the unit e.g. seconds
+     * @return {number} a time in seconds
+     */
+    getStartTimeInSeconds() {
+        let convertedValue = INITIAL_TIME_IN_SECONDS;
+
+        const value = getProp(this.fileStartObj, 'value');
+        const unit = getProp(this.fileStartObj, 'unit');
+
+        if (!value || !unit) {
+            return INITIAL_TIME_IN_SECONDS;
+        }
+
+        if (unit === SECONDS_UNIT_NAME) {
+            convertedValue = parseInt(value, 10);
```

Comment:
Does this mean that we can't start at fractions of a second? Or does Shaka player not support that?

Reply:
Good point. It definitely supports floating point! changing to parseFloat